### DISCUSSION TITLE
Base URL for Superset

### DISCUSSION THREAD

**Discussion by maya-harel** [apache/superset#22125] [Q&A / Help]
Hi

I want to set superset with a baseUrl (http://example.com/superset)

Is there any way to configure this through the superset_config.py ? Without editing the source code

Thanks.

**Comment by sfirke** [comment]
This was asked in the Superset Slack recently: https://apache-superset.slack.com/archives/CK1P3L23E/p1662752446707309.  The answer was:
> while technically possible, it's very difficult to get right. Superset wasn't designed to share a domain, so a dedicated subdomain is your best option
Along with some discussion of what source code modifications are needed.
